pkg/controller/controller_utils_test.go

```diff
@@ -20,6 +20,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
Hm, I wasn't able to unresolve the old comment (not sure why) but this still needs to be addressed